tests/Controller/AbstractApiTestCase.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/** @noinspection PhpUnhandledExceptionInspection */
```

Comment:
This is meta data that probably shouldn't be here?

Reply:
I'm running code inspector on whole project so fixing issues on tests as well.